packages/mc-stats/README.md

```diff
@@ -0,0 +1,37 @@
+# Jetpack Stats
+
+Used to record stats using https://pixel.wp.com/g.gif
+
+## Usage
+
+```php
+$stats = new Automattic\Jetpack\Stats();
```

Comment:
Needs to be updated to the proper class name.

Reply:
done